deltachat-ffi/deltachat.h

```diff
@@ -3445,11 +3444,10 @@ dc_lot_t*        dc_chatlist_get_summary     (const dc_chatlist_t* chatlist, siz
  * @memberof dc_context_t
  * @param context The context object.
  * @param chat_id The chat ID to get a summary for.
- * @param msg_id The message ID to get a summary for.
  * @return The summary as an dc_lot_t object, see dc_chatlist_get_summary() for details.
  *     Must be freed using dc_lot_unref(). NULL is never returned.
  */
-dc_lot_t*        dc_chatlist_get_summary2    (dc_context_t* context, uint32_t chat_id, uint32_t msg_id);
+dc_lot_t*        dc_chatlist_get_summary2    (dc_context_t* context, uint32_t chat_id);
```

Comment:
changing the cffi means we need to discuss this with @r10s because it needs changes not only in desktop, I haven't done it because I wanted to test it inside of the jsornrpc only at first

Reply:
I'd prefer it to be removed at all. There's dc_chatlist_get_summary() which takes dc_chatlist_t, it's more straightforward. We can build a chatlist once and then use it for as long as we need. This way we can avoid "jumping" chat entries while the chatlist is under user's focus. If we do that, DC will be the first messenger solving this stupid annoyance :)

Reply:
`dc_chatlist_get_summary2()` is not used by android/ios for mentioned reasons, so removing it or changing it can be just done.

ftr, `dc_chatlist_get_summary2()` was introduced at #1771 because the chatlist-object was thrown away immediately after creation as it could not be kept in memory easily that time. meanwhile, there is also jsonrpc - maybe things have changed since then?

Reply:
I think we can leave it, but rename to `dc_chat_get_summary()`, it doesn't inflate FFI so much